grc.conf

```diff
@@ -195,7 +195,7 @@ conf.dockerversion
 conf.log
 
 # KubeCTL
-^(^|[/\w\.]+/)kubectl((?!edit|exec|run|go-template).)*$
+(^|[/\w\.]+/)kubectl((?!edit|exec|run|go-template).)*$
```

Comment:
wouldn't

```
^([/\w\.]+\/)?kubectl((?!edit|exec|run|go-template).)*$
```
be a better option?

Reply:
I believe that you are right.
But your change is not only for kubectl but for all commands.

It needs to write a sed command to change all the commands on the last file grc.conf

Reply:
I don't understand why you don't merge this : it is clear that the regular expression of kubectl in /etc/grc.conf is not the same than the others